test/ModelMixins/TableMixinSpec.ts

```diff
@@ -66,7 +66,7 @@ describe("TableMixin", function() {
         ).toEqual([
           "2015-08-07T06:00:00Z",
           "2015-08-07T00:00:00Z",
-          "2015-08-03T06:00:00Z",
+          "2015-08-02T23:59:59Z",
           "2015-08-05T00:00:00Z"
```

Comment:
This test was wrong, this feature only has a single date - so it should have used this

https://github.com/TerriaJS/terriajs/blob/2882e1e90a2d6a27881575bd10c8c2565f324a7b/lib/Table/TableStyle.ts#L34